packages/terra-clinical-item-display/docs/UpgradeGuide.md

```diff
@@ -18,3 +18,4 @@
 * --terra-clinical-item-display-strong-color
 * --terra-clinical-item-display-strong-font-weight
 * --terra-clinical-item-display-strike-through-color
+* --terra-clinical-item-display-primary-color
```

Comment:
Should we include this new themeable var in the changelog as well? If I already pulled this version in, I wouldn't be looking here to cross check the available vars